### ISSUE TITLE
Hot Swapping for js files

### ISSUE THREAD

**Issue by mkraenz** [proSingularity/laguerre#10]
Inside the html embeddings the included js files to build our construction should be reloaded automatically.

**Comment by mkraenz** [comment]
Tried following this [Stackoverflow](http://stackoverflow.com/questions/207513/is-there-a-way-to-hot-swap-javascript-code-within-the-browser) post but did not do what I wanted.

The bookmarklet simply did not work at all.

hotswap.js did not really swap the js file but included a copy of it in the page. Therefore it wasn't compatible with my debugging goals.

**Comment by mkraenz** [comment]
**Conclusions:**

The livereloaded included in grunt-watch simply presses F5 if a watched file has changed.
We will stick with it since the loading time of the geogebra plugin is not as hard as suspected.

On the other hand I was able to use _hotswap.js_ started from a button to switch the js files. Moreover, with grunt-watch not watching .js files, I was able to compile ts files with grunt-watch using `grunt serve` and swap the generated js on buttonclick with hotswap.js.
Unfortunately, the maps and the ts files were not swapped inside the debugger of Chrome, though the js was swapped. Therefore one was unable to see the codechanges in Chrome's debugger. One needed to refresh with F5 to see these changes.
